src/read-time.js

```diff
@@ -1 +1,6 @@
 'use strict';
+
+
+const readTime = function (foo, wpm = 1) {
```

Comment:
Use the function declaration syntax here, `function readTime (foo, wpm = 1)`.

This is a byproduct of the way the files are lining up.
Right now `read-time` is being read AFTER the stuff in `index.js` so `readTime` is undefined when you try to use it in `index.js`.